### ISSUE TITLE
Finding games but not writing files. 

### ISSUE THREAD

**Issue by mscout1** [Emersont1/itchio#70]
When I run this it lists a huge list of files and creates a ton of directories. But there are no files written to any of those directories. Any idea why?

**Comment by kururuchan** [comment]
I can confirm this issue as well. I've run the program as both admin, and not as admin, using CMD and Python on a Windows 10 machine. It also does not create any form of error file.

**Comment by Emersont1** [comment]
Duplicate of #69. Closing and consolidating

**Comment by Emersont1** [comment]
This was an issue with older versions, are you sure you're running 2.2.0?

**Comment by mscout1** [comment]
I'm running what pip brought me, and yep that's 2.2.0. It might be time put out a new version.

**Comment by DebugDax** [comment]
Adding to this, encountering the same issue on 2.2.0.

Running with:
`python -m itchiodl.downloader --api-key=XXX --platform=windows --jobs=4`

Creates subfolders for every item but each folder is empty.

**Comment by mscout1** [comment]
Hi, 

My ADHD decided I am hyper focusing on this fight now. 

The key hitch seems to be in `itchiodl.utils.clean_path`. The `re` module does not have a `replace` function. You probably meant `re.sub`. This raises an `AttributeException` 

The secondary problem was that you never consume generator returned from `ThreadPoolExecutor.map` The results in all the exceptions being swallowed silently.

Once I fixed those, I started running into other exceptions caused by inconsistent handling of file and path names.  I am working on those. 

I hope to have a PR tomorrow. If you don't see one by friday, assume I got board and wandered off.
